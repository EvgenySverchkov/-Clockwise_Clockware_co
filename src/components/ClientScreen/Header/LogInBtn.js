import React from "react";
import { Link } from "react-router-dom";

function LogIn() {
  return (
    <li className="nav-item">
      <Link className="nav-link" to="/client/login">
        Login
      </Link>
    </li>
  );
}

export default LogIn;
