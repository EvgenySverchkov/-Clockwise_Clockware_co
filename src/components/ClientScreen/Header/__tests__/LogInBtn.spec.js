import React from "react";
import {create} from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";

import LogInBtn from "../LogInBtn";

it("Test of <LogInBtn />", ()=>{
    const component = create(
        <MemoryRouter initialEntries={["/"]}>
            <LogInBtn />
        </MemoryRouter>
    );
    expect(component.toJSON()).toMatchSnapshot();
});