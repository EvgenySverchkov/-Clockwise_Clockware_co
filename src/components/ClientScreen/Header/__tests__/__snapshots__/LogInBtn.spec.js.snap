// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of <LogInBtn /> 1`] = `
<li
  className="nav-item"
>
  <a
    className="nav-link"
    href="/client/login"
    onClick={[Function]}
  >
    Login
  </a>
</li>
`;
